profiles/events.py

```diff
@@ -16,16 +16,23 @@ def send_update_events(publisher: EventPublisher) -> Callable[..., None]:
     def event_handler(sender: Flask, changes: tuple) -> None:  # pylint:disable=unused-argument
         ids = []
 
+        LOGGER.info('Processing event(s)')
+
         for instance, operation in changes:  # pylint:disable=unused-variable
+            LOGGER.info('Event: %s - %s', instance, operation)
             if isinstance(instance, Profile):
                 ids.append(instance.id)
             if isinstance(instance, (Affiliation, EmailAddress)):
                 ids.append(instance.profile.id)
 
         for profile_id in set(ids):
             try:
+                LOGGER.info('Sending event for profile id: %s', profile_id)
```

Comment:
Bit inconsistent with the failure message, should be `Sending event for Profile %s`?

Reply:
Changed.